modules/post/windows/gather/credentials/mremote.rb

```diff
@@ -62,7 +63,13 @@ def get_xml(path)
 
   def parse_xml(data)
 
-    mxml= REXML::Document.new(data).root
+    a = File.readlines(data)
+    if a[1].include? "mrng:Connections"
+      a = a.drop(2).join
+    else
+      a = data
+    end
```

Comment:
With the above changes this `else` condition probably won't be needed. I would however add some comments to explain what you are trying to remove here as its not immediately clear from reading the code what is going on.